libraries/chain/db_management.cpp

```diff
@@ -164,6 +164,15 @@ void database::open(
 
       if( !find(global_property_id_type()) )
          init_genesis(genesis_loader());
+      else
+      {
+         _p_core_asset_obj = &get( asset_id_type() );
+         _p_core_dynamic_data_obj = &get( asset_dynamic_data_id_type() );
+         _p_global_prop_obj = &get( global_property_id_type() );
+         _p_chain_property_obj = &get( chain_property_id_type() );
+         _p_dyn_global_prop_obj = &get( dynamic_global_property_id_type() );
+         _p_witness_schedule_obj = &get( witness_schedule_id_type() );
```

Comment:
I get compilation errors on lines 172, 174, 175?!
```
invalid static_cast from type ‘const graphene::db::object’ to type ‘const graphene::chain::chain_property_object&’
invalid static_cast from type ‘const graphene::db::object’ to type ‘const graphene::chain::special_assets_meta_object&’
invalid static_cast from type ‘const graphene::db::object’ to type ‘const graphene::chain::witness_schedule_object&’
```
Apparently need to
```
#include <graphene/chain/chain_property_object.hpp>
#include <graphene/chain/witness_schedule_object.hpp>
#include <graphene/chain/special_authority_object.hpp>
```

Reply:
What's your compiler version? I don't know where to add these header includes. Perhaps you can submit a PR?

Reply:
gcc-5.3.1 - please add them at the top of db_management.cpp